java/code/src/com/suse/manager/maintenance/MaintenanceManager.java

```diff
@@ -528,12 +528,13 @@ public int retractScheduleFromSystems(User user, Set<Long> systemIds) {
      * @param date the date to start looking for maintenance windows
      * @return the resulting list of maintenance windows
      */
-    public List<MaintenanceWindowData> preprocessCalendarData(User user, String operation, Long id, Long date) {
+    public List<MaintenanceWindowData> preprocessCalendarData(User user, String operation, Long id, Long date,
+                                                              Long startOfWeek) {
```

Comment:
Nit: using 64 bits for passing 1 bit of information is a bit overkill :smile: \scnr

Reply:
Haha well yeah you are absolutely right. A boolean would do here as well :sweat:
Dunno what rid me there :rofl: